tools/zetasql_helper/zetasql_helper/local_service/local_service_grpc.h

```diff
@@ -30,6 +30,17 @@ class ZetaSqlHelperLocalServiceGrpcImpl : public ZetaSqlHelperLocalService::Serv
   grpc::Status Tokenize(grpc::ServerContext *context, const TokenizeRequest *req,
                         TokenizeResponse *resp) override;
 
+  grpc::Status ExtractFunctionRange(grpc::ServerContext *context, const ExtractFunctionRangeRequest *request,
+                                    ExtractFunctionRangeResponse *response) override;
+
+  grpc::Status LocateTableRanges(grpc::ServerContext *context,
+                                 const LocateTableRangesRequest *request,
+                                 LocateTableRangesResponse *response) override;
+
+  grpc::Status AllKeywords(grpc::ServerContext *context,
```

Comment:
nit: GetAllKeywordsRequest? method name GetAllKeywords?

Reply:
Yes, I have changed them all to `GetAllKeywords`